config/gradle/utility.gradle

```diff
@@ -22,6 +22,15 @@ ext {
     templatesDir = 'templates'
 }
 
+// Prompts the user to enter a line of text
+def getUserString (String prompt) {
+    println ('\n' + (char)27 + '[1m*** ' + prompt + '\n') // Bold terminal text
```

Comment:
Checked this out as part of running it in a Groovy script on Windows either with regular `cmd` or inside the IntelliJ terminal but neither one worked :(